google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineServerModel.java

```diff
@@ -162,14 +162,20 @@ public void checkConfiguration() throws RuntimeConfigurationException {
           "App Engine facet not found in '" + artifact.getName() + "' artifact");
     }
 
+    if (CloudSdkService.getInstance().getSdkHomePath() == null) {
+      throw new RuntimeConfigurationError(
```

Comment:
Good question! Because without Cloud SDK, we can't have dev_appserver and essentially can't run the run config.

Another good question would be why the above is Warning only. I'm guessing it's because it's still possible to locally run an app without the App Engine facet (which we're currently not using for anything AFAIK).